### PR TITLE
BTA-1553: Content update to 'What is your VAT number' screen

### PR DESCRIPTION
**Pull request by nehakannaujia1**
BTA-1553: Content update to 'What is your VAT number' screen

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -1006,6 +1006,7 @@ whatIsYourVATRegNumber.heading = Beth yw’ch rhif TAW?
 whatIsYourVATRegNumber.error.required = Nodwch rif TAW yn y fformat cywir
 whatIsYourVATRegNumber.error.tooLong = Rydych wedi nodi gormod o rifau
 whatIsYourVATRegNumber.error.tooShort = Nid ydych wedi nodi digon o rifau
+whatIsYourVATRegNumber.hintText = Dyma’r rhif 9 digid ar eich tystysgrif cofrestru ar gyfer TAW. Er enghraifft, '123456789'.
```
Comment (anujakhare) [inline conf/messages.cy]: You have used single quotes with VRN example while it's not showing on page in Welsh.

File: conf/messages.en [hunk 2]
```diff
@@ -1006,6 +1006,7 @@ whatIsYourVATRegNumber.heading = What is your VAT number?
 whatIsYourVATRegNumber.error.required = Enter a VAT number in the correct format
 whatIsYourVATRegNumber.error.tooLong = You have entered too many numbers
 whatIsYourVATRegNumber.error.tooShort = You have not entered enough numbers
+whatIsYourVATRegNumber.hintText = This is the 9-digit number on your VAT registration certificate. For example, '123456789'.
```
Comment (anujakhare) [inline conf/messages.en]: Same case on  English version of page. As per disscussion with @asier-hmrc , Its fine if it does not display on page. Just notice, it is with code but not showing on page.

Comment (chris-ianson) [inline conf/messages.en]: Do the single quotes need to be _smart quotes_

File: conf/messages.cy (hunk 1 above)
Comment (nehakannaujia1) [reply conf/messages.cy]: Yeah i have removed those as its not required.

File: conf/messages.en (hunk 2 above)
Comment (nehakannaujia1) [reply conf/messages.en]: yep removed.

Comment (nehakannaujia1) [reply conf/messages.en]: No we dont need these in text. So i have removed quotes.

**Pull request merged**